src/analyzer/cspParser.js

```diff
@@ -7,6 +7,8 @@ const DIRECTIVES_DISALLOWED_IN_META = [
   "report-uri",
   "sandbox",
 ];
+const ALLOWED_DUPLICATE_KEYS = new Set(["report-uri", "report-to"]);
+export const DUPLICATE_WARNINGS_KEY = "_observatory_duplicate_warnings";
```

Comment:
Nit: How about `duplicate_key_warnings` to be unambiguous?

```suggestion
export const DUPLICATE_WARNINGS_KEY = "_observatory_duplicate_key_warnings";
```